routing/routing_tests/index_graph_test.cpp

```diff
@@ -83,6 +87,24 @@ void TestIngoingEdges(IndexGraph & graph, Segment const & segment,
 }
 
 uint32_t AbsDelta(uint32_t v0, uint32_t v1) { return v0 > v1 ? v0 - v1 : v1 - v0; }
+
+void TestTopologyGraph(TestIndexGraphTopology & graph, TestIndexGraphTopology::Vertex from,
```

Comment:
Probably it's worth moving the first param |graph| to the end of the param list?